cfp.html

```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a not-for-profit event.</li>
```

Comment:
I think `non-profit` is more common expression to describe such type.

Reply:
Oh wonderful, thank you!